jobs/config.json

```diff
@@ -3233,6 +3233,30 @@
       "sig-testing"
     ]
   },
+  "ci-kubernetes-e2e-gce-scale-coredns": {
+    "args": [
+      "--cluster=gce-scale-cluster",
+      "--env=CLUSTER_DNS_CORE_DNS=true",
+      "--env=HEAPSTER_MACHINE_TYPE=n1-standard-8",
```

Comment:
With 3 nodes, we don't need that big heapster machine (we can remove this line).